tasks/sep/030-revert-mt-cannon/NestedSignFromJson.s.sol

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {NestedSignFromJson as OriginalNestedSignFromJson} from "script/NestedSignFromJson.s.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {stdJson} from "forge-std/StdJson.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+import {SystemConfig} from "@eth-optimism-bedrock/src/L1/SystemConfig.sol";
+
+contract NestedSignFromJson is OriginalNestedSignFromJson {
```

Comment:
Could we use the verification contracts instead, like for many Holocene upgrades?

Reply:
Agreed with @sebastianst. 
This is easier for us to review and also increase the consistency for the signers + reviewers. 
I would suggest to use this as example: 
https://github.com/ethereum-optimism/superchain-ops/tree/main/tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain

Reply:
[This one](https://github.com/ethereum-optimism/superchain-ops/blob/dfdb5698166efb334c8146562397b28c5ac53f42/tasks/sep/026-fp-holocene-upgrade-fix/NestedSignFromJson.s.sol#L13-L13) uses the new `DisputeGameUpgrade` verification contract.

Reply:
Discussed more in discord - this was generated via the standard template for setting dispute games.  The verification framework assumes there is permissionless and permissioned game types which isn't always the case so doesn't fit well with the template which can handle all chains.  But for this case I've updated the script to use the verification framework.